client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.0",
+    "passport": "^0.4.0",
+    "react": "^16.6.3",
+    "react-datepicker": "^1.8.0",
+    "react-dom": "^16.6.3",
+    "react-fontawesome": "^1.6.1",
```

Comment:
I think we can use `fontawesome` without using this component

Reply:
If nothing is using fontawesome yet, maybe leave out this dependency? Including fontawesome will add a few hundred kb to the React app, so not worth it unless you really need FA icons.

Reply:
Yeah no worries, removed now.